src/Application/Models/Mint.js

```diff
@@ -0,0 +1,44 @@
+import Denomination from './Denomination';
+
+const denominations = [
```

Comment:
This a small thing and asked purely out of curiosity, but why is this array of denominations not included inside the Mint class, or as a parameter to the constructor?